demos/storybook/stories/app-bar/with-basic-usage.tsx

```diff
@@ -3,17 +3,21 @@ import { number } from '@storybook/addon-knobs';
 import { StoryFnReactReturnType } from '@storybook/react/dist/client/preview/types';
 import { WITH_MIN_PROPS_STORY_NAME } from '../../src/constants';
 import { AppBar } from '@pxblue/react-components';
-import { bodyFiller } from './filler';
+import { Toolbar, Typography } from '@material-ui/core';
+import { getBodyFiller } from '../../src/utils';
 
 export const withBasicUsage = (): StoryFnReactReturnType => {
     const expandedHeight = number('expandedHeight', 200);
     const collapsedHeight = number('collapsedHeight', 64);
+    const scrollThreshold = number('scrollThreshold', 136);
     return (
         <div style={{ display: 'block' }}>
-            <AppBar expandedHeight={expandedHeight} collapsedHeight={collapsedHeight}>
-                <div>Title</div>
+            <AppBar expandedHeight={expandedHeight} collapsedHeight={collapsedHeight} scrollThreshold={scrollThreshold}>
+                <Toolbar>
+                    <Typography variant="h6">Title</Typography>
+                </Toolbar>
             </AppBar>
-            {bodyFiller()}
+            {getBodyFiller()}
```

Comment:
This content is hidden behind the AppBar when it is expanded...that should't be happening.

Reply:
Tried to fix this. Not sure if it's a storybook bug or not. I'll test it elsewhere shortly. As of now the text is no longer hidden but after scrolling below the fold the header scrolls away.